crates/router/src/connector/checkout/transformers.rs

```diff
@@ -1059,7 +1056,7 @@ impl utils::MultipleCaptureSyncResponse for ActionResponse {
     }
 
     fn get_amount_captured(&self) -> Option<i64> {
```

Comment:
can we change this to get_minor_captured_amount

Reply:
Yeah, That is trait function of **MultipleCaptureSyncResponse** and we can change its return type to MinorUnit and we have to change it at all implementations of it